external/fv3fit/fv3fit/pytorch/graph_config.py

```diff
@@ -0,0 +1,64 @@
+import dataclasses
+from typing import Callable
+import torch.nn.functional as F
+import torch
+import torch.nn as nn
+from dgl.nn.pytorch import SAGEConv
+
+
+@dataclasses.dataclass
+class GraphNetworkConfig:
+    """
+    Attributes:
+        in_feats: number of input features
+        out_feats: number of output features
+        n_hidden: maximum number of hidden channels
+        num_step: number of U-net blocks
+        aggregat: type of aggregator, choosen from "mean", "gcn","pool","lstm"
+        activation: activation function
+    """
+
+    in_feats: int = 2
+    out_feats: int = 2
```

Comment:
From last review:

Please use a more verbose name, is this input_features? Shouldn't this value along with output_features be possible to determine algorithmically based on the input and output variable names, by looking at their sizes in the tfdataset? I'm not sure why it's a configuration option here.